addons/mail/static/src/core/common/chat_window_container.js

```diff
@@ -46,11 +38,16 @@ export class ChatWindowContainer extends Component {
         this.chatWindowService = useState(useService("mail.chat_window"));
         this.ui = useState(useService("ui"));
         this.hiddenMenuRef = useRef("hiddenMenu");
+        this.state = useState({ ready: false });
+        this.messaging.isReady.then(() => {
+            // do not wait for messaging to be ready in onWillStart to prevent
+            // from slowing down the whole webclient initial rendering
+            this.state.ready = true;
+        });
```

Comment:
I don't think the extra state is necessary, we already have `this.store.isMessagingReady` in combination with the `isReady` promise.